### ISSUE TITLE
Perlmod does not generate bitfield

### ISSUE THREAD

**Issue by DGA45** [doxygen/doxygen#7495]
For a C-struct using bitfield like this:
```
typedef union
{
  struct
  {
    /** Motor initialized */
    byte _bInitialized          : 1;    
```

the Doxydocs.pm output (PerlMod) does not contain the bitfield description:
```
      public_members => {
        members => [
          {
            kind => 'variable',
            name => '_bInitialized',
            virtualness => 'non_virtual',
            protection => 'public',
            static => 'no',
            brief => {},
            detailed => {
              doc => [
                {
                  type => 'parbreak'
                },
                {
                  type => 'text',
                  content => 'Motor initialized '
                }
              ]
            },
            type => 'byte'
          },
```
while the XML output does:
```
    <compoundname>NVMAPIWLFlags_t._Bits</compoundname>
      <sectiondef kind="public-attrib">
      <memberdef kind="variable" id="structNVMAPIWLFlags__t_8__Bits_1a04d1bce874a071c4ca5c94f518fe44fe" prot="public" static="no" mutable="no">
        <type><ref refid="group__comp__COMMON__public_1ga0c8186d9b9b7880309c27230bbb5e69d" kindref="member">byte</ref></type>
        <definition></definition>
        <argsstring></argsstring>
        <name>_bInitialized</name>
        <bitfield> 1</bitfield>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Motor initialized </para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="NVM_API.h" line="214" column="1"/>
      </memberdef>

```

By looking in the Doxygen source code, we can see the following code in the xmlgen.cpp, function generateXMLForMember:
```
  if (md->memberType()==MemberType_Variable && md->bitfieldString())
  {
    QCString bitfield = md->bitfieldString();
    if (bitfield.at(0)==':') bitfield=bitfield.mid(1);
    t << "        <bitfield>" << convertToXML(bitfield) << "</bitfield>" << endl;
  }
```

This management is missing in the perlmodgen.cpp file, function generatePerlModForMember.

**Comment by albert-github** [comment]
For what purpose do you use the perlmod output (just to get a feeling).

-    Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem? Please don't add external links as they might not be persistent.
-    Please also specify the doxygen version used.

**Comment by DGA45** [comment]
I described our purpose in the #7490 
Here is an example to reproduce the issue:
[Doxy issues example.zip](https://github.com/doxygen/doxygen/files/4035994/Doxy.issues.example.zip)

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).
(before it was accidentally automatically closed)

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.19.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
